dev/snippets/config/templates/stateful_widget.tmpl

```diff
@@ -12,7 +12,7 @@ void main() => runApp(new MyApp());
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return WidgetsApp(
+    return MaterialApp(
```

Comment:
Isn't this now essentially the same as https://github.com/flutter/flutter/blob/master/dev/snippets/config/templates/stateful_widget_material.tmpl?

Reply:
(see also their respective documentation in dev/snippets/config/templates/README.md).

Reply:
I'll revert this file from this PR.  I updated it originally because apps produced by this template were crashing, but I can follow-up with that in a separate PR.

Reply:
In an ideal world we would have tests that apps created from these templates actually are runnable...